commands/previewPush.go

```diff
@@ -129,13 +153,30 @@ func (args *PushArgs) flags() []cli.Flag {
 	return flags
 }
 
+const pwarn = "WARNING: This command will receive a major upgrade in the next release (v4.15). Try the new behavior by using ppreview/ppush. Please test and report any bugs ASAP. The older code will be available as oldpreview/oldpush for a limited number of releases."
+const owarn = "WARNING: oldpreview/preview are temporary commands and may go away in a future release without warning. They are not subject to SemVer. Please upgrade to ppreview/ppush instead."
+
 // Preview implements the preview subcommand.
 func Preview(args PreviewArgs) error {
+	printer.Println(pwarn)
+	return run(args, false, false, printer.DefaultPrinter, nil)
+}
+
+// OldPreview implements the oldpreview subcommand.
+func OldPreview(args PreviewArgs) error {
+	printer.Println(owarn)
 	return run(args, false, false, printer.DefaultPrinter, nil)
 }
 
 // Push implements the push subcommand.
 func Push(args PushArgs) error {
+	printer.Println(pwarn)
+	return run(args.PreviewArgs, true, args.Interactive, printer.DefaultPrinter, &args.Report)
+}
+
+// OldPush implements the push subcommand.
```

Comment:
```suggestion
// OldPush implements the oldpush subcommand.
```

https://github.com/StackExchange/dnscontrol/blob/45b380cbafddb8eb9044c78cc0684769eabbd6df/commands/previewPush.go#L124-L131